apps/builder/app/builder/features/workspace/canvas-tools/outline/label.tsx

```diff
@@ -71,7 +70,16 @@ const LabelContainer = styled(
           bottom: `-${theme.spacing[10]}`,
         },
       },
+      variant: {
+        default: {
```

Comment:
"instance" or "element"?

Reply:
slots are also instances, the only difference is slot/component instance vs everything else